### DISCUSSION TITLE
My rights were not taken care of by the company

### DISCUSSION THREAD

**Discussion by dinhokusanagi** [GitbookIO/community#594] [0. General bugs and questions] [answered]
Delet

**Comment by aikoii** [comment, marked as answer]
Hey @dinhokusanagi, it sounds like you need to connect with the Smart Launcher support team, who host their documentation on our platform. Unfortunately, we can't assist with that as the content and access are managed by the content owner. Thanks!

**Comment by meshalA1** [comment]
انا لم افهم ماهي مشكلتك ولا اعرف هل الكلام موجه لي ام مذا اذا كنت تقصدني ارجو التوضيح ماهي مشكلتك وعن اي موضوع تتحدث
